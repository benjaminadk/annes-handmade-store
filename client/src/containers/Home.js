import React, { Component } from 'react'
import { withStyles } from '@material-ui/core/styles'
import { compose } from 'react-apollo'

const styles = theme => ({
  root: {
    minHeight: '80vh',
    [theme.breakpoints.down('sm')]: {
      minHeight: '10vh'
    }
  },
  logoContainer: {
    display: 'flex',
    justifyContent: 'center',
    marginTop: '1vh'
  },
  logo: {
    width: '35vw',
    height: '35vw',
    [theme.breakpoints.down('sm')]: {
      width: '80vw',
      height: '80vw'
    }
  }
})

class Home extends Component {
  render() {
    const { classes } = this.props
    return (
      <div className={classes.root}>
        <div className={classes.logoContainer}>
          <img
            src="https://s3-us-west-1.amazonaws.com/shopping-site/assets/logo-500x500.svg"
            alt="logo"
            className={classes.logo}
          />
        </div>
      </div>
    )
  }
}

export default compose(withStyles(styles))(Home)
