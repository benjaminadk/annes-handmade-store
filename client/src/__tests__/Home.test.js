import React from 'react'
import Home from '../containers/Home'
import renderer from 'react-test-renderer'

test('Home Page renders correctly', () => {
  const tree = renderer.create(<Home />).toJSON()
  expect(tree).toMatchSnapshot()
})
