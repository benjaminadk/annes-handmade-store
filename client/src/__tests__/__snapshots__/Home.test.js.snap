// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders correctly 1`] = `
<div
  className="Home-root-1"
>
  <h1>
    This site is DEPRECIATED.
  </h1>
  <a
    href="https://anneshandmade.herokuapp.com"
  >
    Checkout the new Anne's Handmade!!!
  </a>
  <div
    className="Home-logoContainer-2"
  >
    <img
      alt="logo"
      className="Home-logo-3"
      src="https://di213806f07x5.cloudfront.net/assets/logo-500x500.svg"
    />
  </div>
  <div
    className="Home-button-4"
  />
</div>
`;
